### PR TITLE
Ii 3473 zipkin logs

### PR DESCRIPTION
**Pull request by gnegri**

### CODE REVIEW

File: zipkin/config.ini [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[zipkin]
```
Comment (yqian10) [inline zipkin/config.ini]: please change name to config.ini.template

File: zipkin/getmetrics_zipkin.py [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+
+    if track['mode'] == "LOG":
+        log_handoff(timestamp, instance, span)
+    else:
+        metric_handoff(timestamp_fixed, span_name, instance, span['duration'])
+
+
+def filter_zipkin_span(service_name, span_name):
+    """ determine if this span should be excluded """
+    if len(agent_config_vars['services_filter']) != 0 and service_name not in agent_config_vars['services_filter']:
```
Comment (yqian10) [inline zipkin/getmetrics_zipkin.py]: 'in' means the partially match. please confirm this is what you want

File: zipkin/getmetrics_zipkin.py [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+
+
+###################################
+# Functions to handle Metric data #
+###################################
+def metric_handoff(timestamp, field_name, instance, data):
+    append_metric_data_to_entry(timestamp, field_name, instance, data)
+    track['line_count'] += 1
+    track['entry_count'] += 1
+    if len(track['current_dict']) >= if_config_vars['chunkLines']:
```
Comment (yqian10) [inline zipkin/getmetrics_zipkin.py]: please convert if_config_vars['chunkLines'] to int

File: zipkin/getmetrics_zipkin.py [hunk 4]
```diff
@@ -0,0 +1,524 @@
...
+    track['current_row'].append(entry)
+    track['line_count'] += 1
+    track['entry_count'] += 1
+    if track['line_count'] >= if_config_vars['chunkLines'] * 100:
+        send_data_wrapper()
+
+
+def prepare_log_entry(timestamp, instance, data):
+    """ creates the log entry """
+    logger.debug(instance + '|' + str(timestamp) + '|' +
```
Comment (yqian10) [inline zipkin/getmetrics_zipkin.py]: add more description to debug mode

File: zipkin/getmetrics_zipkin.py (hunk 2 above)
Comment (gnegri) [reply zipkin/getmetrics_zipkin.py]: yes, should they want to do partial matching.

File: zipkin/getmetrics_zipkin.py (hunk 3 above)
Comment (gnegri) [reply zipkin/getmetrics_zipkin.py]: now converted to int when grabbed from the config file

**Pull request merged**